src/cmd/manage_nsfs.js

```diff
@@ -754,6 +765,50 @@ async function validate_account_args(data, action) {
     }
 }
 
+// we take the data object and extract its properties according to the schema:
+// - we unwrap ths sensitive string (that we have from fetch_account_data)
+// - we change the creation_date to string
+function extract_properties_from_account_data_and_validate_schema(data) {
+    try {
+        const account_to_validate = {
+            ...data,
+            name: data.name?.unwrap(),
+            email: data.email?.unwrap(),
+            creation_date: data.creation_date?.toString(),
+            access_keys: [{
+                access_key: data.access_keys && data.access_keys.length && data.access_keys[0]?.access_key?.unwrap(),
+                secret_key: data.access_keys && data.access_keys.length && data.access_keys[0]?.secret_key?.unwrap(),
+            }],
+            nsfs_account_config: {
+                ...data.nsfs_account_config,
+                distinguished_name: data.nsfs_account_config?.distinguished_name?.unwrap(),
+            },
+        };
+        nsfs_schema_utils.validate_account_schema(account_to_validate);
+    } catch (err) {
+        dbg.error('extract_properties_from_account_data_and_validate_schema: schema validation error:', err);
+        throw err;
```

Comment:
this will throw internal error, please add a new CLI error specifying the details of the error

Reply:
Will add, thanks.